### PR TITLE
#462

### PR DESCRIPTION
**Pull request by mpuhacz**
**Story / Bug id:** https://github.com/stxnext-csr/volontulo/issues/462
**Reference person:** @mpuhacz
**Description:**
- Reorganized imports.
- Email backend settings.
- New buttons: accept and delete instead of edit and show.
- Two new class based views: OffersDelete, OffersAccept

**Unit Tests:**
- `apps/volontulo/tests/views/test_offers.py`

### CODE REVIEW

**Comment by magul** [comment]
```
$ pep8 --exclude='apps/volontulo/migrations/*,node_modules,.ropeproject' .
./apps/volontulo/admin.py:15:1: W391 blank line at end of file
./apps/volontulo/tests/views/test_offers.py:224:80: E501 line too long (86 > 79 characters)
./apps/volontulo/tests/views/test_offers.py:233:80: E501 line too long (86 > 79 characters)
./apps/volontulo/tests/views/test_offers.py:242:80: E501 line too long (86 > 79 characters)
./apps/volontulo/tests/views/test_offers.py:251:80: E501 line too long (86 > 79 characters)
./apps/volontulo/tests/views/test_offers.py:334:80: E501 line too long (86 > 79 characters)
./apps/volontulo/tests/views/test_offers.py:343:80: E501 line too long (86 > 79 characters)
./apps/volontulo/tests/views/test_offers.py:352:80: E501 line too long (86 > 79 characters)
./apps/volontulo/tests/views/test_offers.py:361:80: E501 line too long (86 > 79 characters)
./apps/volontulo/views/offers.py:18:80: E501 line too long (80 > 79 characters)
./apps/volontulo/views/offers.py:304:80: E501 line too long (89 > 79 characters)
./apps/volontulo/views/offers.py:323:80: E501 line too long (89 > 79 characters)
./apps/volontulo/views/offers.py:332:1: E303 too many blank lines (3)
```

**Comment by magul** [comment]
```
$ pylint --load-plugins pylint_django --min-similarity-lines=10 apps
Messages
--------
+-----------------+------------+
|message id       |occurrences |
+=================+============+
|locally-disabled |116         |
+-----------------+------------+
|invalid-name     |6           |
+-----------------+------------+
|no-member        |4           |
+-----------------+------------+
|unused-import    |1           |
+-----------------+------------+
```

**Comment by magul** [comment]
Rest is available in travis console.

File: apps/volontulo/templates/admin/list_offers.html [hunk 1]
```diff
@@ -42,17 +42,44 @@
                     </div>
                 </td>
                 <td class="text-right">
-                    <a href="{% url 'offers_edit' offer.title|slugify offer.id  %}" class="btn btn-info">
-                        <span aria-hidden="true" class="glyphicon glyphicon-edit"></span> Edycja
+                    <a class="btn btn-success accept-offer"  data-toggle="modal" href="{% url 'offers_accept' offer.id  %}">
```
Comment (tomaszwozniak) [inline apps/volontulo/templates/admin/list_offers.html:45]: extra space after offer"

File: apps/volontulo/tests/views/test_offers.py [hunk 2]
```diff
@@ -142,6 +142,236 @@ def test_offers_list_for_admin(self):
...
+
+    @classmethod
+    def setUpTestData(cls):
+        u"""Set up data for all tests."""
+        cls.organization = Organization.objects.create(
+            name=u'',
+            address=u'',
+            description=u'',
+        )
+        cls.organization.save()
```
Comment (tomaszwozniak) [inline apps/volontulo/tests/views/test_offers.py]: unnecessary save()

File: apps/volontulo/tests/views/test_offers.py [hunk 3]
```diff
@@ -142,6 +142,236 @@ def test_offers_list_for_admin(self):
...
+            'offer_status': 'unpublished',
+            'recruitment_status': 'closed',
+            'action_status': 'ongoing',
+        }
+
+        cls.inactive_offer = Offer.objects.create(
+            status_old='NEW',
+            **common_offer_data
+        )
+        cls.inactive_offer.save()
```
Comment (tomaszwozniak) [inline apps/volontulo/tests/views/test_offers.py]: ditto

File: apps/volontulo/tests/views/test_offers.py [hunk 4]
```diff
@@ -142,6 +142,236 @@ def test_offers_list_for_admin(self):
...
+        cls.active_offer = Offer.objects.create(
+            status_old='ACTIVE',
+            **common_offer_data
+        )
+        cls.active_offer.save()
```
Comment (tomaszwozniak) [inline apps/volontulo/tests/views/test_offers.py]: ditto (check whole diff)

File: apps/volontulo/tests/views/test_offers.py [hunk 5]
```diff
@@ -142,6 +142,236 @@ def test_offers_list_for_admin(self):
...
+
+        volunteer_user = User.objects.create_user(
+            u'volunteer@example.com',
+            u'volunteer@example.com',
+            u'123volunteer'
+        )
+        volunteer_user.save()
+        cls.volunteer = UserProfile(user=volunteer_user)
```
Comment (tomaszwozniak) [inline apps/volontulo/tests/views/test_offers.py:189]: objects.create

**Pull request merged**